src/storage/v2/durability/snapshot.cpp

```diff
@@ -2597,6 +2599,501 @@ RecoveredSnapshot LoadSnapshotVersion18or19(const std::filesystem::path &path, u
   return {info, recovery_info, std::move(indices_constraints)};
 }
 
+RecoveredSnapshot LoadSnapshotVersion20or21(const std::filesystem::path &path, utils::SkipList<Vertex> *vertices,
```

Comment:
Here we should manually test that we can recover from the previous Memgraph version, namely v2.22.1